File: Upcoder/Khối 11/Chương 6 Xâu/Xâu lớn hơn/K11_XauLonHon.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    string a, b;
    getline(cin , a);
    getline(cin , b);
    if(a > b)
    cout << a;
    else if(b > a)
    cout << b;
    else 
    cout <<"Hai xau bang nhau";
    return 0;
}

File: Upcoder/Khối 11/Chương 6 Xâu/Xâu lớn hơn/K11_XauLonHon.cs
//Code này chỉ đc 3.75 Điểm 
using System;
namespace Xaulonhon
{
    public class Program
    {
        static void Main(string[]args)
        {
            string A = Console.ReadLine();
            string B = Console.ReadLine();
            if (string.Compare(A, B) > 0)
                Console.Write(A);
            else if (string.Compare(A, B) < 0)
                Console.Write(B);
            else
                Console.Write("Hai xau bang nhau");
        }
    }
}
